docs/sources/panels-visualizations/visualizations/flame-graph/index.md

```diff
@@ -18,55 +18,77 @@ weight: 850
 
 # Flame graph panel
 
-The flame graph takes advantage of the hierarchical nature of profiling data. It condenses data into a format that allows you to easily see which code paths are consuming the most system resources.
+Flame graph panel is a visualization that allows you to visualize profiling data. It can represent the profile as a flame graph, table or both
```

Comment:
```suggestion
Flame graph panel is a visualization that allows you to visualize profiling data. It can represent the profile as a flame graph, table, or both
```